src/epic.rs

```diff
@@ -0,0 +1,61 @@
+/// An Epic object contains all the information about an epic poem that we care
+/// about when we're generating messages.
+#[derive(Debug)]
+pub struct Epic {
+    /// The title of the epic poem
+    pub title: &'static str,
+    /// We store the Wikipedia article that we want to use for attribution. If a
+    /// given poem doesn't have an appropriate Wikipedia article to link to,
+    /// this will contain `None`.
+    pub wiki: Option<&'static str>,
```

Comment:
It might be a good idea to add some sort of method that returns the appropriate Wikipedia article based on the wiki name. I've checked and it seems that all wiki pages have start with "https://en.wikipedia.org/wiki/" and then the title of the wiki page. 

Note that spaces are changed to underscores and some titles have non-ASCCI characters 
(For example, Lục Vân Tiên which has the article https://en.wikipedia.org/wiki/L%E1%BB%A5c_V%C3%A2n_Ti%C3%AAn)

Reply:
The unescaped titles seem to work fine in twitter, so I think I'll only do the underscore transform.